ddl/reorg.go

```diff
@@ -37,7 +37,7 @@ import (
 	"github.com/pingcap/tidb/util/logutil"
 	"github.com/pingcap/tidb/util/mock"
 	"github.com/pingcap/tidb/util/ranger"
-	"github.com/pingcap/tipb/go-tipb"
+	"github.com/wangwangwar/tipb/go-tipb"
```

Comment:
You can learn from https://github.com/pingcap/parser#step-2-make-your-parser-changes-take-effect-in-tidb-and-run-ci to change the tipb dependences.